tools/devops/automation/build-cronjob.yml

```diff
@@ -7,124 +7,38 @@
 #
 parameters:
 
-- name: provisionatorChannel
-  displayName: Provisionator channel to use 
-  type: string
-  default: 'latest'
-
-- name: pool
-  type: string
-  displayName: Bot pool to use
-  default: automatic
-  values:
-  - pr
-  - ci
-  - automatic
-
-- name: runTests
-  displayName: Run Simulator Tests
+- name: runGovernanceTests
   type: boolean
   default: true
 
-- name: runDeviceTests
-  displayName: Run Device Tests 
-  type: boolean
-  default: false
-
-- name: runOldMacOSTests
-  displayName: Run Tests on older macOS versions 
-  type: boolean
-  default: true
+- name: stageDisplayNamePrefix
+  type: string
+  default: ''
 
-- name: runWindowsIntegration
-  displayName: Run Windows integration tests
-  type: boolean
-  default: true
+- name: dependsOn
+  type: string
+  default: ''
 
-- name: runGovernanceTests
-  displayName: Run Governance Checks
-  type: boolean
-  default: true
+- name: dependsOnResult
+  type: string
+  default: ''
 
-- name: runSamples
-  displayName: Run Samples
+- name: isPR
   type: boolean
   default: false
```

Comment:
We do know is not a PR because is a pipeline with no trigger. We can remove it and just pass false where needed.